index.html

```diff
@@ -129,7 +129,7 @@
         <div class="follow-us" align="center"> 
 <a class="fa fa-facebook social-icon" href="https://www.facebook.com/MozilliansTamilNadu" target="_href"></a> 
 <a class="fa fa-twitter social-icon" href="https://twitter.com/mozillaTN" target="_href"></a> 
-<a class="fa fa-google-plus social-icon" href="#" target="_href"></a> </div>
+<a class="fa fa-google-plus social-icon" href="https://plus.google.com/+MozillaindiaOrg" target="_href"></a> </div>
 </footer>

```

Comment:
Sorry for the mistake, I will rectify it as soon as possible
On 19 Feb 2016 4:36 pm, "Tummala Dhanvi (c0mrad3)" <notifications@github.com>
wrote:

> In index.html
> <https://github.com/MozillaTN/Weeks-Of-Contirbution/pull/64#discussion_r53448240>
> :
>
> > @@ -129,7 +129,7 @@
> >          <div class="follow-us" align="center">
> >  <a class="fa fa-facebook social-icon" href="https://www.facebook.com/MozilliansTamilNadu" target="_href"></a>
> >  <a class="fa fa-twitter social-icon" href="https://twitter.com/mozillaTN" target="_href"></a>
> > -<a class="fa fa-google-plus social-icon" href="#" target="_href"></a> </div>
> > +<a class="fa fa-google-plus social-icon" href="https://plus.google.com/+MozillaindiaOrg" target="_href"></a> </div>
> >  </footer>
>
> @drunkskunk <https://github.com/drunkskunk> the link you have added is
> Mozilla India but this website is about Mozilla Tamil Nadu
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MozillaTN/Weeks-Of-Contirbution/pull/64/files#r53448240>
> .
>